### DISCUSSION TITLE
Lack of harmonised data from INSPIRE

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#30] [General]
Despite the passed INSPIRE deadline of 21 October 2020, requiring the provision of datasets fully conformant to the INSPIRE requirements, there are still data gaps. When trying to use alternative data to INSPIRE, namely national/regional data with their own data schema (meanwhile the missing INSPIRE data is available), it is needed to have a fully documented Data Model (DM) and Data Dictionary (DD) to allow data usability. Therefore, the data providers should provide DM and DD together with this “alternative” data.

**Comment by naomithiru** [comment]
There are indeed many instances of available metadata but unavailable download services from the INSPIRE portal encountered.

**Comment by AndersOstmanNovogit** [comment]
This issue is reported in section 4.2.1 in D4.3

The problem seems to be that the existing data do not have proper documentation, for instance a documented data model and/or data dictionary. How did you solve the problem? I see a couple of different possible solutions. They all rely on that you have some knowledge about the terminology being used and/or the cartographic rendering rules.
a) Asking the data provider for additional information
b) Reverse engineering (creating a DM/DD a posteriori) by using column/field names, type of geometry and domain knowledge about the phenomena.
c) Reverse engineering using printed map with a map legend and/or interpretation of aerial photographs

**Reply by naomithiru** [reply]
In the instances where there was no available harmonized data from INSPIRE(both data and corresponding DD), we used data from different sources for example, local(city/municipalities) data portals. For the tracing use-case this issue was mostly faced in finding sewer datasets. 
-  In this case, solution (b) above was used, with knowledge of the subject matter and terminologies helping us navigate a dataset.
- Additionally, we used sewer data from the UWWTD (Urban Waste Water Treatment Directive), which has a well documented DD

**Reply by AndersOstmanNovogit** [reply]
Now reported in D4.3 section 4.3.1

**Comment by MayteTDGeograma** [comment]
In PopImpact we also used solution (b) above, where subject knowledge and terminologies helped us to navigate through a dataset.
